values.yaml

```diff
@@ -261,8 +261,8 @@ persistence:
     disableredirect: false
     # Specify the "caBundleSecretName" if the storage service uses a self-signed certificate.
     # The secret must contain keys named "ca.crt" which will be injected into the trust store
-    # of registry's containers.
-    # caBundleSecretName:
+    # of registry's and chartmuseum's containers.
```

Comment:
please remove `and chartmuseum's` because chartmuseum will be removed from harbor-helm 1.12.0 and later version

Reply:
Fixed in [`517fe8a` (#1434)](https://github.com/goharbor/harbor-helm/pull/1434/commits/517fe8adfa9c4f6ba7788f55359a188a42c1fe24)